src/main/java/ulcrs/controllers/UlcController.java

```diff
@@ -0,0 +1,64 @@
+package ulcrs.controllers;
+
+import spark.Request;
+import spark.Response;
+import spark.RouteGroup;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+
+import static spark.Spark.before;
+import static spark.Spark.get;
+
+public class UlcController extends BaseController {
+
+    @Override
+    public RouteGroup routes() {
+        return () -> {
+            before("/*", (request, response) -> log.info("endpoint: " + request.pathInfo()));
+            get("/login", this::loginTest);
+        };
+    }
+
+    private String loginTest(Request request, Response response) {
+
+        URL url;
+        URLConnection urlConn;
+        BufferedReader br = null;
+        String line;
+
+        StringBuilder page = new StringBuilder();
+
+        try {
+            url = new URL("https://tbrtest.engr.wisc.edu/");
```

Comment:
It's actually on dropin-dev.engr.wisc.edu right now. Might be better to make this a class constant.

Reply:
Yeah we could probably do that. I believe this was merged into backend as test code, so I was just going with it to change it once we worked on future tickets that required change

Reply:
Since this test code isn't the main focus of this PR anyway. this is fine.